src/WebJobs.Script.Grpc/Abstractions/IRpcServer.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Rpc
+{
+    public interface IRpcServer
+    {
+        void Start();
+
+        Task ShutdownAsync();
+
+        Uri Uri { get; }
```

Comment:
>Uri [](start = 8, length = 3)

- What's a sample Uri? Where do we get it?

Reply:
something like `grpc://127.0.0.1:50051`

I'm interested in the protocol (grpc vs other transport), host, and port. They're passed to the workers so they know what to connect to.